superset/security/manager.py

```diff
@@ -736,7 +736,14 @@ def create_missing_perms(self) -> None:
 
         logger.info("Fetching a set of all perms to lookup which ones are missing")
         all_pvs = set()
-        for pv in self.get_session.query(self.permissionview_model).all():
+        for pv in (
+            self.get_session.query(self.permissionview_model)
+            .options(
+                eagerload(self.permissionview_model.permission),
+                eagerload(self.permissionview_model.view_menu),
+            )
+            .all()
+        ):
```

Comment:
One small thing, I see this is effectively same as the change in the other location, except here you're using `self.permissionview_model` and there you're using the name directly. I think using `self.permissionview_model` is the right way to go based on patterns I've seen elsewhere, but seems we probably need to stay DRY if we've already done it two different ways, and especially since the `eagerload` stuff is a bit fiddly.

Maybe just separate it into a `def _get_all_pvms` or similar? Gives us an opportunity to mention that we're doing it eagerly to avoid too many queries that way too.

Reply:
Done.